.github/workflows/check.yml

```diff
@@ -18,10 +18,8 @@ jobs:
         env:
           MAPBOX_DOWNLOADS_TOKEN: ${{ secrets.MAPBOX_DOWNLOADS_TOKEN }}
 
-      - name: Run All Tests using Fastlane
-        run: |
-          bundle install
-          fastlane test_all
+      - name: Run All Tests
+        run: ./Scripts/./test.sh
```

Comment:
Like above, I think you could remove the `./` in the middle and it would look like this `./Scripts/test.sh` :wink:

Reply:
here 7605cbe9eed6324568ac98188ee2faa9ea2c2c70